spec/features/idv/analytics_spec.rb

```diff
@@ -47,6 +47,38 @@
       ),
     }
   end
+  let(:gpo_path_events) do
+    common_events = {
```

Comment:
Could we do something like this to avoid the duplication?

```suggestion
    common_events = happy_path_events.merge(
      'IdV: USPS address letter requested' => { enqueued_at: Time.zone.now },
    )
```